__tests__/index.test.js

```diff
@@ -48,6 +50,88 @@ describe("GET /api/articles", () => {
         });
       });
   });
+  test("status 200 - returns articles sorted by default created at desc", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(articles).toBeSortedBy("created_at", {
+          descending: true,
+        });
+      });
+  });
+  test("status 200 - returns articles sorted by created_at ordered asc ", () => {
+    return request(app)
+      .get("/api/articles?order=asc")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(articles).toBeSortedBy("created_at", {
+          descending: false,
+        });
+      });
+  });
+  test("status 200 - returns articles sorted by votes: def desc", () => {
+    return request(app)
+      .get("/api/articles?sort_by=votes")
```

Comment:
- [ ] TEST -  could you check that you can both sort_by=&order= in a query?

Just to check it sorts, and changes the default as well.